remote/templates/search.html

```diff
@@ -0,0 +1,15 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:spacer {"height":"32px"} -->
+<div style="height:32px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+<!-- wp:search {"label":"Search","width":100,"widthUnit":"%","buttonText":"Search","style":{"border":{"radius":"0px"}}} /-->
```

Comment:
I wonder if this should be added as a pattern also, so that the label and text can be translated.

Reply:
Good idea, I moved that code to a pattern a made those strings translatable.